envi/archs/arm/__init__.py

```diff
@@ -11,9 +11,11 @@
 class ArmModule(envi.ArchitectureModule):
 
     def __init__(self, name='armv6'):
+        import envi.archs.thumb16.disasm as eatd
```

Comment:
I only allow "late imports" like this to combat dependency loop problems.  Is this one of those?